app/service_invite/rest.py

```diff
@@ -31,36 +31,25 @@
 def create_invited_user(service_id):
     request_json = request.get_json()
     invited_user = invited_user_schema.load(request_json)
-    save_invited_user(invited_user)
-
+    is_join_request = request_json.get("is_join_request")
     if invited_user.service.has_permission(BROADCAST_TYPE):
         template_id = current_app.config["BROADCAST_INVITATION_EMAIL_TEMPLATE_ID"]
+    elif is_join_request:
+        template_id = current_app.config["JOIN_LIVE_SERVICE_REQUESTER_APPROVED_TEMPLATE_ID"]
     else:
         template_id = current_app.config["INVITATION_EMAIL_TEMPLATE_ID"]
 
     template = dao_get_template_by_id(template_id)
-    service = Service.query.get(current_app.config["NOTIFY_SERVICE_ID"])
 
-    saved_notification = persist_notification(
-        template_id=template.id,
-        template_version=template.version,
-        recipient=invited_user.email_address,
-        service=service,
-        personalisation={
-            "user_name": invited_user.from_user.name,
-            "service_name": invited_user.service.name,
-            "url": invited_user_url(
-                invited_user.id,
-                request_json.get("invite_link_host"),
-            ),
-        },
-        notification_type=EMAIL_TYPE,
-        api_key_id=None,
-        key_type=KEY_TYPE_NORMAL,
-        reply_to_text=invited_user.from_user.email_address,
-    )
-
-    send_notification_to_queue(saved_notification, queue=QueueNames.NOTIFY)
+    if is_join_request:
```

Comment:
given this flow's so different

"create an invite in the database, send that user an email with a link to join service"

"vs person has already been added to service, give them a confirmation email saying so"

i feel like these could be separate endpoints?

that would make this a lot tidier and much easier to reason about when looking at eg access logs